paddle/fluid/imperative/prepared_operator.h

```diff
@@ -164,6 +139,8 @@ class PreparedOp {
            const NameVarMap<VariableWrapper>& outs,
            const framework::AttributeMap& attrs);
 
+  const framework::OpKernelType& kernel_type() const { return kernel_type_; }
```

Comment:
kernel_type()  -> KernelType()

Reply:
according to the code style, kernel_type is more recommended, 

![image](https://user-images.githubusercontent.com/22561442/103883214-e6033680-5117-11eb-8c98-0d9a96f770f5.png)